### PR TITLE
feat(versions): Add version history item actions dropdown

### PR DESCRIPTION
**Pull request by jstoffan**
Version Actions

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jstoffan has signed the CLA. Thanks for the pull request!

File: src/elements/content-sidebar/versions/VersionsItemActions.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        background-color: $off-white;
+        fill: $fours;
+    }
+
+    .fill-color {
+        fill: $nines;
+    }
+
+    .icon-clock-past {
+        margin-left: -2px;
```
Comment (ConradJChan) [inline src/elements/content-sidebar/versions/VersionsItemActions.scss]: What is this margin adjustment used for?

File: src/elements/content-sidebar/versions/VersionsItemActions.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                )}
+                {showPromote && (
+                    <MenuItem className="bcs-VersionsItemActions-item" onClick={onPromote}>
+                        <IconUpload {...ICON_SIZE} />
+                        <FormattedMessage {...messages.versionActionPromote} />
+                    </MenuItem>
+                )}
+                {showRestore && (
+                    <MenuItem className="bcs-VersionsItemActions-item" onClick={onRestore}>
+                        <IconClockPast height={14} width={14} />
```
Comment (ConradJChan) [inline src/elements/content-sidebar/versions/VersionsItemActions.js]: Does this icon need to be tweaked to be consistent with the others? (Not of the scope of this PR but just wondering)

File: src/elements/content-sidebar/versions/VersionsItemActions.scss (hunk 1 above)
Comment (jstoffan) [reply src/elements/content-sidebar/versions/VersionsItemActions.scss]: The clock icon has a slightly different size and viewbox than the other icons, so it needed a little extra love.

File: src/elements/content-sidebar/versions/VersionsItemActions.js (hunk 2 above)
Comment (jstoffan) [reply src/elements/content-sidebar/versions/VersionsItemActions.js]: Ideally, all our icons would have the same default dimensions, padding, and viewbox, but that's not the case currently. I know @tjgupta has put some effort into this area, but my understanding is that it's a pretty large effort from both us and design.

**Pull request merged**